kong/plugins/circuit-breaker/handler.lua

```diff
@@ -120,11 +120,11 @@ function CircuitBreakerHandler:init_worker()
 			local route_id = key_parts[3]
 
 			if route_id ~= "" then
-				circuit_breakers:remove_breakers_by_level(route_id) -- Route level circuit breaker
+				circuit_breakers:remove_breakers_by_group(route_id) -- Route level circuit breaker
 			elseif service_id ~= "" then
-				circuit_breakers:remove_breakers_by_level(service_id) -- Service level circuit breaker
+				circuit_breakers:remove_breakers_by_group(service_id) -- Service level circuit breaker
 			else
-				circuit_breakers:remove_breakers_by_level("global") -- Global circuit breaker
+				circuit_breakers:remove_breakers_by_group("global") -- Global circuit breaker
```

Comment:
https://github.com/dream11/lua-circuit-breaker/blob/v1.0.2/src/factory.lua#L58-L70

Reply:
Thanks, will have a look and merge